internal/v1/handler_blueprints.go

```diff
@@ -405,13 +405,13 @@ func (h *Handlers) GetBlueprint(ctx echo.Context, id openapi_types.UUID, params
 	return ctx.JSON(http.StatusOK, blueprintResponse)
 }
 
-func (h *Handlers) lintBlueprint(ctx echo.Context, blueprint *BlueprintBody, fixup bool) ([]BlueprintLintItem, error) {
+func (h *Handlers) lintBlueprint(ctx echo.Context, blueprint *BlueprintBody, fixup bool, blueprintEntry *db.BlueprintEntry) ([]BlueprintLintItem, error) {
```

Comment:
Please do not add a parameter that carries the same information: blueprintEntry already contains `Body` so there is no reason to pass `blueprint *BlueprintBody` anymore.

I highly suggest to this refactoring in completely isolated commit - this makes it so much easier not only to review but it also helps with testing and git history. Once this is done, you can make use of the newly introduced parameter in the next commit immediately.